packages/stratimikos/CMakeLists.txt

```diff
@@ -6,6 +6,16 @@ TRIBITS_ADD_ENABLE_TEUCHOS_TIME_MONITOR_OPTION()
 
 TRIBITS_ADD_EXPLICIT_INSTANTIATION_OPTION()
 
+# Original behavior was to include ThyraEpetraAdapters as a required dependency.
+# Modify this to make sure that ThyraEpetraAdapters is enabled if Epetra is, and
+# that ThyraTpetraAdapters is enabled if Tpetra is
+IF(${PACKAGE_NAME}_ENABLE_Epetra )
+  SET(${PACKAGE_NAME}_ENABLE_ThyraEpetraAdapters ON)
```

Comment:
Does this logic actually work?  I don't believe setting `${PROJECT_NAME}_ENABLE_<anything>=ON` has any effect (at at least not the effect you want) when you set this in a `<packageDir>/CMakeLists.txt`.  See [Full TriBITS Project Configuration](https://tribits.org/doc/TribitsDevelopersGuide.html#full-tribits-project-configuration).